java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -283,14 +284,15 @@ public void sadd_srem_scard_smembers_key_with_non_set_value(BaseClient client) {
         String key = UUID.randomUUID().toString();
         assertEquals(OK, client.set(key, "foo").get());
 
-        Exception e = assertThrows(ExecutionException.class, () -> client.sadd(key, "bar").get());
+        Exception e =
+                assertThrows(ExecutionException.class, () -> client.sadd(key, new String[] {"bar"}).get());
```

Comment:
Try avoid using strings which could be names
```suggestion
                assertThrows(ExecutionException.class, () -> client.sadd(key, new String[] {"pewpew"}).get());
```